### PR TITLE
Show PCI devices in SystemMonitor

### PR DESCRIPTION
**Pull request by bugaevc**

### CODE REVIEW

File: Applications/SystemMonitor/main.cpp [hunk 1]
```diff
@@ -258,3 +262,57 @@ GWidget* build_file_systems_tab()
...
+    pci_widget->set_layout(make<GBoxLayout>(Orientation::Vertical));
+    pci_widget->layout()->set_margins({ 4, 4, 4, 4 });
+    auto* pci_table_view = new GTableView(pci_widget);
+    pci_table_view->set_size_columns_to_fit_content(true);
+
+    auto db = PCIDB::Database::open();
+
+    Vector<GJsonArrayModel::FieldSpec> pci_fields;
+    pci_fields.empend(
+        "Placement", TextAlignment::CenterLeft,
```
Comment (bugaevc) [inline Applications/SystemMonitor/main.cpp]: Is this a good column name? I'm not terribly familiar with PCI, so this was the best i could come up with.

**Comment by bugaevc** [comment]
image

File: Applications/SystemMonitor/main.cpp (hunk 1 above)
Comment (awesomekling) [reply Applications/SystemMonitor/main.cpp]: I would call this "Address". That would also match the `PCI::Address` kernel object :)

File: Applications/SystemMonitor/main.cpp [hunk 2]
```diff
@@ -89,6 +91,8 @@ int main(int argc, char** argv)
 
     tabwidget->add_widget("File systems", build_file_systems_tab());
 
+    tabwidget->add_widget("PCI Devices", build_pci_devices_tab());
```
Comment (awesomekling) [inline Applications/SystemMonitor/main.cpp]: "PCI devices" to be consistent with "File systems"

**Pull request merged**